### PR TITLE
Overview of changes (current and potential) since original validation

### PR DESCRIPTION
**Pull request by Aastedet**
Moved from #69

### CODE REVIEW

File: vignettes/changes-since-original.Rmd [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    diabetes type (due to increasing use in treatment of non-diabetes).
+2.  Diabetes type reclassification based on insulin purchases in the
+    previous year is no longer used.
+
+## Future potential changes
+
+1.  Add support for using medical birth register to define pregnancies
+    to censor GDM. Allows censoring GLD purchases all the way back to
+    1995 (rather than 1997 onward, as the obstetric codes are limited
+    to), and extends the window of valid dates of diagnosis to 1996
```
Comment (signekb) [inline vignettes/changes-since-original.Rmd:25]:
```suggestion
    to), and enable the extension of the window of valid dates of diagnosis to 1996
```

**Review by signekb — COMMENTED** [review]
Looks good! I have added some suggestions :)

**Review by lwjohnst86 — APPROVED** [review]
I made some mostly formatting changes, as well as small grammatical edits. I also converted most passive voice ("it was done") sentences to active voice ("we did").

**Comment by lwjohnst86** [comment]
Since this has had several rounds of edits, I will merge in.

**Pull request merged**